### ISSUE TITLE
Minor: logging of queries after final string interpolation

### ISSUE THREAD

**Issue by bcattle** [coleifer/peewee#1014]
Hey, I love this library. The [docs](http://docs.peewee-orm.com/en/latest/peewee/database.html#logging-queries) say you can turn on logging of queries with 

    # Print all queries to stderr.
    import logging
    logger = logging.getLogger('peewee')
    logger.setLevel(logging.DEBUG)
    logger.addHandler(logging.StreamHandler())

This produces output such as the following:

    ('
              SELECT count(DISTINCT username)
                FROM get_rows_created_days_ago_with_lookback(NULL::stats_daily_active_users, %(days_prior)s, %(lookback_days)s)
            ', {'days_prior': 1, 'lookback_days': 7})

In other words, it's printing the output before the final string interpolation step. 

By accessing the psycopg2 `cursor.query` attribute, you are able to get the output _after_ that final string interpolation step, returning:

    SELECT count(DISTINCT username)
        FROM get_rows_created_days_ago_with_lookback(NULL::stats_daily_active_users, 1, 7)

Would it be clearer if we change the logging code to use the psycopg2 `cursor.query` property? Thanks.

**Comment by coleifer** [comment]
It would be much better! If I recall correctly, you can also use psycopg2.mogrify to get the final query. Unfortunately pysqlite doesnt implement this:

* http://bugs.python.org/issue9506
* https://github.com/ghaering/pysqlite/issues/20#issuecomment-26258837

Looks like mysql-python doesn't implement this either.

So, because it's just postgres, I think I'll favor lowest common denominator here.
